olp-cpp-sdk-core/tests/olpclient/OlpClientTest.cpp

```diff
@@ -45,62 +45,13 @@ class NetworkMock : public olp::http::Network {
   MOCK_METHOD(void, Cancel, (olp::http::RequestId id), (override));
 };
 
-class Client : public ::testing::TestWithParam<bool> {
+class OlpClientTest : public ::testing::Test {
  protected:
   olp::client::OlpClientSettings client_settings_;
   olp::client::OlpClient client_;
 };
 
-INSTANTIATE_TEST_SUITE_P(TestOffline, Client, ::testing::Values(false));
-
-class ClientDefaultAsyncHttp : public Client {};
-INSTANTIATE_TEST_SUITE_P(TestOnline, ClientDefaultAsyncHttp,
-                         ::testing::Values(true));
-
-TEST_P(ClientDefaultAsyncHttp, GetGoogleWebsite) {
```

Comment:
What happened to these tests? Will they be moved in a new PR?

Reply:
Yes, online tests don't belong here. I'll bring them back shortly after this commit.